examples/Example2_OLED_Distance/Example2_OLED_Distance.ino

```diff
@@ -80,7 +80,7 @@ void loop()
   myUltrasonic.getDistance(distance);
 
   // Convert distance, which is an integer, to char so that we can print it.
-  snprintf(distanceBuff, 4, "%d", distance);
+  snprintf(distanceBuff, 6, "%d", distance);
```

Comment:
The number of bytes (now 6) cannot be bigger than the array size (still 4). If you need to display more characters, make the array bigger too.

Reply:
Agreed, here is the error I was seeing at the time: 
![image](https://github.com/sparkfun/SparkFun_Qwiic_Ultrasonic_Arduino_Library/assets/8891606/f310b2b2-9e98-4d55-a6b1-2be1f33eb80a)
 Let look into it a bit more before I change it back.